example-mlem-get-started/generate.sh

```diff
@@ -0,0 +1,149 @@
+#!/bin/bash
+# See https://dvc.org/doc/tutorials/get-started
+
+# Setup script env:
+#   e   Exit immediately if a command exits with a non-zero exit status.
+#   u   Treat unset variables as an error when substituting.
+#   x   Print commands and their arguments as they are executed.
+set -eux
+
+HERE="$( cd "$(dirname "$0")" ; pwd -P )"
+REPO_NAME="example-mlem-get-started"
+
+BUILD_PATH="$HERE/build"
+
+pushd $BUILD_PATH
+if [ ! -d "$BUILD_PATH/.venv" ]; then
+  exit 1
+  virtualenv -p python3 .venv
+  export VIRTUAL_ENV_DISABLE_PROMPT=true
+  source .venv/bin/activate
+  echo '.venv/' > .gitignore
+  pip install gitpython
```

Comment:
Why you need to install gitpython?

Reply:
not sure)))))

Reply:
this is related to the way DVC calculates it's version in the dev env - https://dvc.org/doc/install/pre-release (add hash to the end of the version). If MLEM does the same (and it should be) then you might also need this. I would check if DVC has changed this mechanism though.

Reply:
thanks, created https://github.com/iterative/example-repos-dev/pull/111 and removed it from the docs (pending redeploy)